src/utils/components/hover.ts

```diff
@@ -74,7 +74,9 @@ const hover = ({ reporter }: { reporter: TelemetryReporter | null }) => {
       content.supportHtml = true;
       content.isTrusted = true;
       content.supportThemeIcons = true;
-      reporter?.sendTelemetryEvent("hover");
+      setTimeout(() => {
```

Comment:
Does this ever get cleared in case the user removes the hover?

Reply:
It does. 

To test this I renamed the event to "strictHover", hovered a few lines and immediately removed the hover, and saw if the telemetry tracked the event or not. It didn't.